src/main/java/bisq/desktop/components/paymentmethods/SameBankForm.java

```diff
@@ -51,37 +51,4 @@ public SameBankForm(PaymentAccount paymentAccount, AccountAgeWitnessService acco
                         GridPane gridPane, int gridRow, BSFormatter formatter, Runnable closeHandler) {
         super(paymentAccount, accountAgeWitnessService, inputValidator, gridPane, gridRow, formatter, closeHandler);
     }
-
-    @Override
-    public void updateAllInputsValid() {
-        boolean result = isAccountNameValid()
-                && inputValidator.validate(bankAccountPayload.getHolderName()).isValid
-                && paymentAccount.getSingleTradeCurrency() != null
-                && ((CountryBasedPaymentAccount) paymentAccount).getCountry() != null;
-
-        final String countryCode = bankAccountPayload.getCountryCode();
-        if (countryCode != null) {
-            if (BankUtil.isBankNameRequired(countryCode))
-                result = result && inputValidator.validate(bankAccountPayload.getBankName()).isValid;
-
-            if (BankUtil.isBankIdRequired(countryCode))
-                result = result && inputValidator.validate(bankAccountPayload.getBankId()).isValid;
-
-            if (BankUtil.isBranchIdRequired(countryCode))
-                result = result && inputValidator.validate(bankAccountPayload.getBranchId()).isValid;
-
-            if (BankUtil.isAccountNrRequired(countryCode))
-                result = result && inputValidator.validate(bankAccountPayload.getAccountNr()).isValid;
-        }
-        allInputsValid.set(result);
-    }
-
-    @Override
-    protected void addHolderNameAndIdForDisplayAccount() {
-        Tuple2<Label, TextField> tuple = addLabelTextField(gridPane, ++gridRow, Res.getWithCol("payment.account.owner"));
-        TextField holderNameTextField = tuple.second;
-        holderNameTextField.setMinWidth(300);
-        holderNameTextField.setText(bankAccountPayload.getHolderName());
-    }
-
```

Comment:
@ManfredKarrer Is there a reason for these overrides in the Same Bank Form case?

Reply:
Hm.. don't remember why it was overridden. Just test it to see if all works as expected.